node/resolvers/Queries/Users.ts

```diff
@@ -220,6 +220,60 @@ export const listUsers = async (
   }
 }
 
+const getRoleAndPermissionsByEmail = async ({
+  email,
+  module,
+  skipError = false,
+  ctx,
+}: {
+  email: string
+  module: string
+  skipError: boolean
+  ctx: Context
+}) => {
+  let ret = {
+    role: {
+      id: '',
+      name: '',
+      slug: '',
+    },
+    permissions: [],
+  }
+
+  if (!email) return ret
+
+  const userData: any = await getUserByEmail(null, { email }, ctx)
+
+  if (!userData.length && !skipError) {
+    throw new Error('User not found')
+  }
+
+  if (!userData.length) return ret
+
+  const userRole: any = await getRole(null, { id: userData[0].roleId }, ctx)
+
+  if (!userRole && !skipError) {
+    throw new Error('Role not found')
+  }
+
+  if (!userRole) return ret
+
```

Comment:
There are a couple of throw errors through the code but without a catch. 
So, I'd put a try/catch on withUserPermissions logging it.
<img width="662" alt="image" src="https://user-images.githubusercontent.com/5812946/161336122-0d874bfd-3156-42f9-9eac-cd8e7be9b5d3.png">